atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1591,6 +1591,11 @@ void WebContents::TabTraverse(bool reverse) {
   web_contents()->FocusThroughTabTraversal(reverse);
 }
 
+v8::Local<v8::Object> WebContents::CreateEvent() {
+  return mate::internal::CreateJSEvent(isolate(), GetWrapper(), nullptr,
```

Comment:
Can you expose this method via event module, `atom/browser/api/event.{cc|h}`, its an internal module. Also better to reduce the number of `_*` method we have on public apis.

Reply:
@deepak1556 what about re-using the `v8util`?

Reply:
`v8util` exposes methods that can be used in both browser and render process. Since this event creation is specific to browser process, I would prefer not adding it there.

Reply:
ok, exposed new `event` browser-only internal module